README.md

```diff
@@ -28,9 +29,9 @@ basic details are as follows:
 7. Once a release enters LTS, no new features may be added to that release.
    Changes are limited to bug fixes, security updates, documentation updates,
    certain performance improvements that can be demonstrated to not break
-   existing applications, etc. Semver-minor changes are only permitted if
-   required for bug fixes. Semver-major changes are only permitted if required
-   for critical security and bug fixes.
+   existing applications, and possible npm updates, etc. Semver-minor changes
```

Comment:
Suggest reordering this list because I initially read it as "performance improvements that can be demonstrated to not break existing applications and possible npm updates" (wut?). Just pulling the npm bit earlier in the list would help with readability I think.